src/python/__init__.py

```diff
@@ -65,5 +65,6 @@
 
     register_v0_v4_envs()
     register_v5_envs()
-except ImportError:
+except ImportError as e:
+    print(e)
```

Comment:
Remove, currently, I would have it silently error if `Gymnasium` hasn't been installed otherwise a more helpful error rather than a weird printed exception when loading ale-py